tests/test-data/dv_order_api_post_supply_delivery.json

```diff
@@ -51,7 +51,7 @@
   "extension": [
     {
       "url": "http://joinallofus.org/fhir/tracking-status",
-      "valueString": "enroute"
+      "valueString": "in_transit"
```

Comment:
perhaps we should make this upper case to ensure we're catching it correctly everywhere.

Reply:
PTSC is sending it to us lowercase, so I think we should probably keep it lowercase here.